crates/turbopack/src/module_options/mod.rs

```diff
@@ -231,6 +226,13 @@ impl ModuleOptionsVc {
             .collect(),
         );
 
+        let ecmascript_options = EcmascriptOptions {
+            split_into_parts: enable_tree_shaking,
+            import_parts: enable_tree_shaking,
+            transforms: app_transforms,
+            compile_time_info,
```

Comment:
@sokra Do you have any idea how can I get `compile_time_info` from here?

Reply:
~Move `compile_time_info` from AssetContext/ModuleAssetContext to `ModuleOptionsContext`.~

That's not that easy.

* Add `compile_time_info` to `ModuleOptionsContext`
* Add a TODO that `compile_time_info` should be removed from `AssetContext`

Reply:
At here, I need a way to get `CompileTimeInfoVc`, not `Option<CompileTimeInfoVc>`